### ISSUE TITLE
Question regarding MAE Pretraining

### ISSUE THREAD

**Issue by LeoXinhaoLee** [yossigandelsman/test_time_training_mae#3]
Hi, thanks so much for releasing code of this inspiring paper. However, I notice that in your ``engine_pretrain.py`` model, the training loss is ``loss = 0.01 * loss_dict["classification"] + loss_dict["mae"]``, in which using the supervised classification loss seems to differ from the pure SSL training paradigm of the MAE paper. I'm wondering if I've missed something. Thank you very much for you help.

**Comment by yossigandelsman** [comment]
You are absolutely correct. This is a leftover from a joint-training experiment, and I just removed it now. To be sure, you can always use the original pre-training phase from the [MAE code](https://github.com/facebookresearch/mae), or their [model](https://dl.fbaipublicfiles.com/mae/pretrain/mae_pretrain_vit_large_full.pth), which I used.
